app/schemas.py

```diff
@@ -272,6 +272,8 @@ def handle_template_merge(self, in_data):
             if in_data['template']['template_type'] == 'email':
                 in_data['subject'] = template.replaced_subject
             in_data.pop('personalisation', None)
+        else:
+            in_data['body'] = in_data['template']['content']
```

Comment:
Sorry for not spotting this when I reviewed the PR yesterday.

You could get rid of the `if in_data.get('personalisation'):` entirely and it would work the same.